sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -158,13 +147,32 @@ public Session createSession(@Nullable String defaultSchema) {
         private Map<String, Portal> portals = new HashMap<>();
         private String lastPortalName;
 
-
         private Session(Executor executor, TransportKillJobsNodeAction transportKillJobsNodeAction, String defaultSchema) {
             this.executor = executor;
             this.transportKillJobsNodeAction = transportKillJobsNodeAction;
             this.defaultSchema = defaultSchema;
         }
 
+        public UUID jobId() {
```

Comment:
please pass these things into the Portal once it is created if it is required inside. This stuff shouldn't be exposed outside.

The portal should be a private thing of a Session and stuff that is needed by the Portal shouldn't be exposed public on the session